src/main/java/com/datastax/oss/protocol/internal/request/Batch.java

```diff
@@ -104,6 +106,9 @@ protected static int computeFlags(int serialConsistency, long defaultTimestamp,
     if (keyspace != null) {
       flags = Flags.add(flags, ProtocolConstants.QueryFlag.WITH_KEYSPACE);
     }
+    if (nowInSeconds != Integer.MIN_VALUE) {
```

Comment:
@olim7t Is it the accepted convention for unset flag values in native protocol to be initialized to things like `Integer.MIN_VALUE`, as opposed to declaring a global constant that can be referenced? I only ask because I was looking at the Driver PR and see that the driver has to know the convention so that the flag value isn't accidentally set to some number that will be processed as an actual value, and not simply unset.

I see that it is already done for other things (i.e. `defaultTimestamp` is considered unset if its value is `Long.MIN_VALUE`), so I assume that this is the convention.

Reply:
I've added a commit to extract two constants.

Overall native-protocol is pretty barebones: no getters, not a lot of checks, and basically you have to read the code to figure out how everything works. This has worked so far for our internal use, but we might have to revisit it at some point if it gets more external use.

Reply:
:+1: